spec/controllers/distributions_controller_spec.rb

```diff
@@ -263,8 +263,54 @@
           expect(flash[:notice]).to eq("Distribution updated!")
           expect(flash[:error]).to be_nil
           expect(flash[:alert]).to be_nil
+
+        end
+      end
+
+      context "verify totals in 'Total in Category Diapers'" do
+        let(:item) { create(:item, name: "Diapers", organization: organization) }
+        let(:storage_location) { create(:storage_location, organization: organization) }
+      
+        before(:each) do
+          TestInventory.create_inventory(organization, {
+            storage_location.id => {
+              item.id => 20
+            }
+          })
+        end
+        let(:distribution) { create(:distribution, storage_location: storage_location) }
+        let(:params) do
+          {
+            organization_name: organization.id,
+            id: distribution.id,
+            distribution: {
+              storage_location_id: distribution.storage_location.id,
+              line_items_attributes:
+                {
+                  "0": { item_id: item.id, quantity: 15 },
+                }
+            }
+          }
+        end  
+              
+        it "ensures the totals in 'Total in Category Diapers' match" do
```

Comment:
This spec doesn't seem to test the fix. It's testing the `create` method, not the `index` method.

You should be making a request spec for this, not a controller spec.